runtime/shared_common/shrinit.cpp

```diff
@@ -3516,6 +3516,7 @@ j9shr_init(J9JavaVM *vm, UDATA loadFlags, UDATA* nonfatal)
 		}
 
 		config->getCacheSizeBytes = j9shr_getCacheSizeBytes;
+		config->isSharedClassEnabledByDefault = j9shr_isSharedClassEnabledByDefault;
```

Comment:
The field is `getSharedClassCacheMode` and the function is `j9shr_getSharedClassCacheMode `.